ballerina-tests/tests/utils.bal

```diff
@@ -168,6 +168,9 @@ isolated function validateCompleteMessage(websocket:Client wsClient, string id =
 
 isolated function validateConnectionClousureWithError(websocket:Client wsClient, string expectedErrorMsg) {
     json|error response = readMessageExcludingPingMessages(wsClient);
-    test:assertTrue(response is error);
-    test:assertEquals((<error>response).message(), expectedErrorMsg);
+    if response is error {
+        test:assertEquals((<error>response).message(), expectedErrorMsg);
```

Comment:
No need of casting here.
```suggestion
        test:assertEquals(response.message(), expectedErrorMsg);
```

Reply:
As I checked type narrowing doesn't work here

Reply:
Updated

Reply:
Seems like. this isn't updated properly.

Reply:
changes pushed